scripts/build.d/vim

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRCSYNC=${SRCSYNC:-tarball}
+
+pkgname=vim
+pkgbranch=${VERSION:-master}
+pkgfull=$pkgname
+
+# unpack (clone)
+syncgit https://github.com/${pkgname} ${pkgname} ${pkgbranch} ${pkgfull}
```

Comment:
However, here you choose to unconditionally get the source by using git. It is inconsistent to what you write in line # 5.

Please use a tarball by default.  Pulling from git is more time-consuming, less stable, and should serve as a secondary approach.

Reply:
@yungyuc
I see. However, the official repo of vim doesn't provide md5sum of a specific tarball file.
Should I download it and compute by myself (which I think is not a good practice)?
Or perhaps there's actually a md5sum coming along with a specific tarball file that I just did not know?

![image](https://user-images.githubusercontent.com/24734750/220350638-4d22a5f7-6115-4db3-a312-94bccb785fe1.png)

Reply:
I got it! I will update the script soon.